### PR TITLE
Remove debug globals

### PR DESCRIPTION
**Pull request by murrant**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: app/Http/Controllers/DeviceController.php [hunk 1]
```diff
@@ -11,8 +11,11 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\Auth;
 use LibreNMS\Config;
+use LibreNMS\Util\Debug;
 use LibreNMS\Util\Graph;
 use LibreNMS\Util\Url;
+use const false;
+use const true;
```
Comment (Jellyfrog) [inline app/Http/Controllers/DeviceController.php:18]: What does these do?

File: includes/common.php [hunk 2]
```diff
@@ -427,27 +425,6 @@ function del_dev_attrib($device, $attrib_type)
     return DeviceCache::get((int) $device['device_id'])->forgetAttrib($attrib_type);
 }
 
-/*
- * convenience function - please use this instead of 'if ($debug) { echo ...; }'
- */
-if (! function_exists('d_echo')) {
```
Comment (Jellyfrog) [inline includes/common.php:433]: This is used a bit here and there

File: LibreNMS/Util/Debug.php [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+use Illuminate\Database\Events\QueryExecuted;
+use LibreNMS\DB\Eloquent;
+use Log;
+
+class Debug
+{
+    private static $debug = false;
+    private static $verbose = false;
+
+    public static function set($debug = true, bool $silence = false): bool
```
Comment (Jellyfrog) [inline LibreNMS/Util/Debug.php:38]: Why not typehint debug?

File: app/Http/Controllers/DeviceController.php (hunk 1 above)
Comment (murrant) [reply app/Http/Controllers/DeviceController.php:18]: stupid intellij put those in there...  I'll remove them.

File: includes/common.php (hunk 2 above)
Comment (murrant) [reply includes/common.php:433]: @Jellyfrog it is a duplicate function. There is still an implementation.

File: LibreNMS/Util/Debug.php (hunk 3 above)
Comment (murrant) [reply LibreNMS/Util/Debug.php:38]: I did at first, but we are supplying this function to an on boot startup, and the first argument sent to the function is an application.  So, the options are change that, make a specific function, or make this less strict.   I went with the later and added the cast below.

**Review by Jellyfrog — APPROVED** [review]
I'm all for removing globals

**Pull request merged**